static/templates/subscription_creation_form.handlebars

```diff
@@ -45,6 +45,14 @@
                             {{/tr}}
                         </label>
                     </li>
+                    <li>
+                        <label class="radio">
+                            <input type="radio" name="privacy" value="invite-only-public-history" />
+                            {{#tr this}}
+                            <b>Private with public message history:</b> must be invited by a member; new members can also see messages sent before they joined; hidden from non-administrator users
```

Comment:
Maybe "Private with history: must be invited by a member; new members can view complete message history; ..."